### PR TITLE
{bio}[foss/2018b] R-bundle-Bioconductor v3.8

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic95 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ba19444719d3c49000d2cec7ecfb5e9d for a full test report.

File: easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.8-foss-2018b-R-3.5.1.eb [hunk 1]
```diff
@@ -0,0 +1,987 @@
...
+        'https://bioconductor.org/packages/3.8/data/annotation/src/contrib/',
+        'https://bioconductor.org/packages/3.8/data/experiment/src/contrib/',
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz'
+}
+ext_options = exts_default_options
+bioconductor_options = exts_default_options
```
Comment (boegel) [inline easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.8-foss-2018b-R-3.5.1.eb]: @smoors Both these lines can be removed safely, they're never used?

File: easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.8-foss-2018b-R-3.5.1.eb [hunk 2]
```diff
@@ -0,0 +1,987 @@
...
+    ('S4Vectors', '0.20.1', {
+        'checksums': ['ef379329429f638f954d76c023d8509a8476bcbc824e067517c8767b0ccf90a3'],
+    }),
+    ('IRanges', '2.16.0', {
+        'checksums': ['81d9a0687665f6dacb970190266358c3ac1046e86d55c49e38b18760a6be5752'],
+    }),
+    ('GenomeInfoDbData', '1.2.0', {
+        'checksums': ['ac8c370b85021786ef67d9fdd1872930368abd2a46593147320e7b7d31b52636'],
+    }),
+    ('bitops', '1.0-6', {
```
Comment (boegel) [inline easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.8-foss-2018b-R-3.5.1.eb]: @smoors Why are you including R packages here that are already included in the `R` easyconfig that is used as a dependency?

Comment (smoors) [reply easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.8-foss-2018b-R-3.5.1.eb]: Good question.. I think this is what the easy_update tool spitted out. Should have checked this more carefully.

Comment (verdurin) [reply easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.8-foss-2018b-R-3.5.1.eb]: Yes, that's almost certainly an `easy_update` artifact. The upstream uses combined `easyconfigs` that include both in one, I believel.

**Comment by akesandgren** [comment]
@smoors Any updates on this?

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic19 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ac0e52b105c31f5f7298b4cc703d0f20 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @smoors!

**Pull request merged**